analyzer/src/test/kotlin/managers/GoModTest.kt

```diff
@@ -70,4 +73,59 @@ class GoModTest : WordSpec({
             id.toVcsInfo().revision shouldBe "v43.3.0"
         }
     }
+
+    "parseWhyOutput" should {
+        "detect packages that are used" {
+            val output = listOf(
```

Comment:
If we assume [1] as output format then blank lines should be added.
Same below.

[1] https://golang.org/ref/mod#go-mod-why

Reply:
The current implementation is actually a bit more robust regarding empty lines. So the tests cover both cases, existing and missing empty lines.

Reply:
Maybe it's a nit: I still believe the input format according to above link does contain blank lines, so this test is now testing implementation details it shouldn't be testing.s Even worse, that implementation detail is tested multiple times redundantly.
As consequence, a valid refactoring may break test which should not be the case.